src/main/java/mate/academy/Main.java

```diff
@@ -4,7 +4,6 @@
 import mate.academy.listeners.SampleListener;
 
 public class Main {
-    // Feel free to play with this main method during the task implementation
     public static void main(String[] args) {
         EventManager manager = new EventManager();
```

Comment:
The instantiation of 'EventManager' is missing the 'new' keyword. It should be 'new EventManager()' to create a new instance.